### DISCUSSION TITLE
Color RGB "style=""background-color: rgba(0, 0, 0, 0)"" can be translated by #000000 ?

### DISCUSSION THREAD

**Discussion by jeremievy** [mganss/HtmlSanitizer#429] [Q&A]
Hello, 

i use AngleSharp.Css.Values.Color.UseHex= true; 

input  : background-color: rgba(0, 0, 0, 0)
output  : background-color: rgba(0, 0, 0, 0)

i would like get  #000000 int the output.

do you have a suggest ?

https://dotnetfiddle.net/qhpHLN

**Comment by mganss** [comment]
This currently works only if the alpha value is 1.0.

> /// This will not be applied in case of transparent colors, i.e.,
        /// when alpha is not 1.

https://github.com/AngleSharp/AngleSharp.Css/blob/15deaee3586d4c37d0054bc80d579ed70f7c74f6/src/AngleSharp.Css/Values/Primitives/Color.cs#L376-L381

**Comment by mganss** [comment]
See https://github.com/AngleSharp/AngleSharp.Css/issues/132
